packages/traceability/issuer-api/src/pods/certificate/handlers/transfer-certificate.handler.ts

```diff
@@ -64,17 +64,23 @@ export class TransferCertificateHandler implements ICommandHandler<TransferCerti
         }
 
         try {
-            await onChainCert.transfer(
+            const transferTx = await onChainCert.transfer(
                 to,
                 BigNumber.from(amount ?? onChainCert.owners[from]),
                 from
             );
+
+            const receipt = await transferTx.wait(1);
```

Comment:
What's that wait(1)? It looks creepy

Reply:
Wait for 1 transaction confirmation from the miner.

Reply:
For a sec I thought it's like 1ms or something :D Thanks god